app/controllers/crypto_coin.js

```diff
@@ -25,10 +27,29 @@ exports.listCoins = (req, res, next) => {
   const { id, currency } = req.user;
   return getCoinsByUser(id)
     .then(coins => convertCoins(coins, currency))
-    .then(coins => cryptoList(coins))
+    .then(coins => serializeCoins(coins))
     .then(coins => {
       logger.info('Finish list coins');
-      return res.send({ coins });
+      return res.send({ coins, currency });
+    })
+    .catch(next);
+};
+
+exports.topCoins = (req, res, next) => {
+  logger.info('Start list coins');
+  const { id, currency } = req.user;
+  const { ord } = req.query;
+  return getCoinsByUser(id)
+    .then(coins => convertCoins(coins, currency))
+    .then(convCoins => {
+      const serialCoins = serializeCoins(convCoins);
+      const orderCoins = orderArrayDesc(serialCoins, 'price');
```

Comment:
You can do a single method that orders depending on the value that is passed as a parameter. And then take the top 3